src/app/js/public/getCustomPage.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import translate from 'redux-polyglot/translate';
+import fetch from 'fetch-with-proxy';
+
+import { polyglot as polyglotPropTypes } from '../propTypes';
+
+export class CustomPage extends Component {
+    state = {};
+    UNSAFE_componentWillMount() {
+        fetch(`/customPage${this.props.link}`)
```

Comment:
Why the `$` ? Why not just use a query param or just `/customPage/page`?

Reply:
:trollface: